### DISCUSSION TITLE
TRPC Version 10.3/10.4 changed type from date to string.

### DISCUSSION THREAD

**Discussion by martin-fv** [trpc/trpc#3381] [Q&A]
The latest TRPC update broke my storybook script https://github.com/trpc/trpc/discussions/1879#discussioncomment-3846341

It now expects a string instead of a date

![Screenshot from 2022-12-12 19-36-04](https://user-images.githubusercontent.com/105156564/207035581-00839a9a-8aba-477a-8f09-1ce5fd4d847b.png)

toJSDate throws an error as it expects a string.

More details coming soon...

**Comment by KATT** [comment]
Are you using a data transformer? If not, this looks right.

If you're using a transformer, I think you have to use `inferTransformedProcedureOutput` from `@trpc/server/shared` in order for it to infer correctly here
